plugins/modules/vmware_guest_info.py

```diff
@@ -308,7 +308,7 @@ def main():
                 if not HAS_VSPHERE:
                     module.fail_json(msg="Unable to find 'vCloud Suite SDK' Python library which is required."
                                          " Please refer this URL for installation steps"
-                                         " - https://code.vmware.com/web/sdk/60/vcloudsuite-python")
+                                         " - https://code.vmware.com/web/sdk/6.5/vsphere-automation-python")
```

Comment:
```suggestion
                                         " - https://code.vmware.com/web/sdk/vsphere-automation-python")
```

I think it would be better to use the generic link to the vSphere Automation SDK for Python. Not the one specifically linking to version 6.5.

Apart from this: Thanks for the PR :-)